src/cultionet/models/lightning.py

```diff
@@ -494,24 +522,25 @@ def configure_optimizers(self):
             max_iter=self.max_iter,
             line_search_fn=None
         )
-        lr_scheduler_lbfgs = lr_scheduler.ReduceLROnPlateau(
-            optimizer_lbfgs,
-            factor=0.1,
-            patience=5
+        model_lr_scheduler = lr_scheduler.CosineAnnealingLR(
```

Comment:
So you import `lr_scheduler` as `from torch.optim import lr_scheduler`. You also have `lr_scheduler: str = 'CosineAnnealingLR',`  as an argument in `CultioLitModel(LightningModule):`
Those will not conflict, right?

Reply:
Great catch -- I'll fix this.